lms/djangoapps/commerce/utils.py

```diff
@@ -109,3 +117,247 @@ def upgrade_url(self, user, course_key):
             else:
                 return reverse('verify_student_upgrade_and_verify', args=(course_key,))
         return None
+
+
```

Comment:
Note: A lot of the following is moving the code to this utils file to match the new access method.